taskwiki/viewport.py

```diff
@@ -58,6 +67,8 @@ def __init__(self, line_number, cache, tw,
             constants.DEFAULT_SORT_ORDER
         )
 
+        self.count = count and int(count) or -1
```

Comment:
The purpose of this boolean expression is not entirely clear to me. Maybe we could just have the default value of `count` set as None?

Reply:
I change the default to `None`. The boolean `and` is needed to convert the count to an integer. `int(None)` raises an error. Originally, I set the default to `-1` to be sure that the count is an integer.